examples/asrc_demo/src/usb/usb_audio.h

```diff
@@ -5,6 +5,8 @@
 #ifndef USB_AUDIO_H_
 #define USB_AUDIO_H_
 
+#include "avg_buffer_level.h"
```

Comment:
added include, but no other changes in this file where you use it. This line can be removed